pymc_extras/statespace/models/structural/core.py

```diff
@@ -136,6 +141,7 @@ class StructuralTimeSeries(PyMCStateSpace):
            methods (2nd ed.). Oxford University Press.
     """
 
+    # TODO need to discuss cutting some of these args down. All the <foo>_names args are already inside of <foo>_info
```

Comment:
Yep let's get rid of them. The `info` should be the single source of truth for all information about the model.